contracts/cw-proposal-single/src/msg.rs

```diff
@@ -191,4 +191,7 @@ pub enum QueryMsg {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-pub struct MigrateMsg {}
+pub enum MigrateMsg {
+    FromBeta,
+    FromCompatible,
```

Comment:
just to make sure i understand FromCompatible is for a `v1` DAO that just missed these updates?

Reply:
from compatible is intended for any non-state changing code migration!